test/ui-test/testSuites/global_shared/steps/startupSteps.py

```diff
@@ -0,0 +1,85 @@
+from utils.FileManager import *
+from common.Common import *
+
+from screens.StatusWelcomeScreen import StatusWelcomeScreen
+from screens.StatusMainScreen import StatusMainScreen
+from screens.StatusChatScreen import StatusChatScreen
+
+# Project settings properties:
+_status_desktop_app_name = "nim_status_client"
+_status_data_folder_path = "../../../../../Status/data"
+_status_fixtures_folder_path = "../../../fixtures/"
+_status_shared_images_path = "../shared/searchImages/"
+_status_qt_path = "../../../../../Status/qt"
+_app_closure_timeout = 2 #[seconds]
+
+# Test context properties names:
+_aut_name = "aut_name"
+_status_data_folder = "status_data_folder_path"
+_fixtures_root = "fixtures_root"
+_search_images = "search_images"
+_scenario_name = "scenario_name"
+
+# Test static objects creation:
+_welcome_screen = StatusWelcomeScreen()
+_main_screen = StatusMainScreen()
+_chat_screen = StatusChatScreen()
+
+# Static class used to define init / startup static methods outside bdd context that can be reused in different `hooks` as well as in specific bdd `steps` files.
+class StartupSteps:
```

Comment:
you don't need a class if all members of the class are @staticmethod,
just declare the function

Reply:
That's true.. I will change it! Thanks